types/joi-password-complexity/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-class": ["allow-constructor-only"]
```

Comment:
Remove the rules here and fix errors if any.. We want new packages to extend the default rules and fix all the errors

Reply:
@sheetalkamat Thanks for your feedback. I tried to remove that rule but how do I define a type for a class that only has a constructor then? It says I should use a function instead but I am not sure what is meant by that.

Reply:
You can add `// tslint:disable-next-line:no-unnecessary-class` to the line before class declaration instead of disabling it for the whole package.